### ISSUE TITLE
Qwik is planing an official auth solution using auth.js

### ISSUE THREAD

**Issue by bbigras** [wmalarski/qwik-next-auth-example#9]
Just a heads-up.

https://discord.com/channels/842438759945601056/842438761287254019/1074126920037453874

**Comment by wmalarski** [comment]
Yeah, I know. I am really looking forward to this. I saw Manu mentioning this on Twitter about a month ago - just after auth.js first release. I hacked this version based on solid.js adapter, super curious how the official version will look like.

**Comment by bbigras** [comment]
https://github.com/BuilderIO/qwik/commit/dd4fd92422540a6f6facbc1ea13e2ad5f3b1a150

**Comment by wmalarski** [comment]
It's super interesting that `loader$` and `action$` functions are used inside of another function and it's fine. And those functions are called in some random file. This is from docs:
>> Loaders can only be declared inside the src/routes folder, in a layout.tsx or index.tsx file, and they MUST be exported.

I guess things are changing quickly:) For sure this will open space for a lot of exploration.
